sass/directives/_carousels.scss

```diff
@@ -105,7 +105,7 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  padding: #{$base-margin--small} #{($base-padding / 3)};
```

Comment:
@MKwenhua I think this would make a lot more sense / be a lot easier if you didn't condense the padding calls onto one line, please split this line out so it is more readable in the future